app/src/main/kotlin/com/hippo/ehviewer/gallery/ArchivePageLoader.kt

```diff
@@ -161,7 +165,17 @@ class ArchivePageLoader(
         false
     }
 
-    override fun preloadPages(pages: List<Int>, pair: Pair<Int, Int>) {}
+    override fun preloadPages(pages: List<Int>, bounds: Pair<Int, Int>) {
```

Comment:
Does preload pages for Archive actually make sense?
We preload pages just for reduce time user wait for network request, but decode is fast and mostly costs less than 1 seconds, should we waste CPU decoding an image may not be shown?

Reply:
I do think the preload mechanism of LazyList is enough, if we ought to preload it ourselves, we must decay preload range then.

Reply:
Well, sort of. Decompression may take a measurable time depending on the format.
And we scroll 0.75 * viewport height at a time, so a few pages need to be preloaded to avoid loading while scrolling.

Reply:
Meanwhile, unlike network request, both decompress & decode is **not** cancellable, I feel like there will be something wrong during fast scroll across pages.

Reply:
I think we could hardcode preload range to +-2 pages given user have normal read speed.

Reply:
For pager modes, even ±1 would be enough.
For webtoon modes, it becomes a little complicated as we don't know 0.75 * viewport height = how many pages.

Reply:
For a 20:9 screen, 20 / 9 * 0.75 / 1.4125 ≈ 1.2.
So we'll need to preload 2 pages at times (when more than 0.8 of the next page is in screen).
And we need to consider 16:9 CGs, 20 / 9 * 0.75 * 16 / 9 ≈ 3.

Reply:
> For a 20:9 screen, 20 / 9 * 0.75 / 1.4125 ≈ 1.2. So we'll need to preload 2 pages at times (when more than 0.8 of the next page is in screen). And we need to consider 16:9 CGs, 20 / 9 * 0.75 * 16 / 9 ≈ 3.

Then let's hardcode range to 3, just not let it reach maximum 17